transports/http/endpoints/api/merkleroots/model.go

```diff
@@ -0,0 +1,54 @@
+package merkleroots
+
+import (
+	"github.com/libsv/bitcoin-hc/domains"
+)
+
+// merkleRootConfirmationResponse is a confirmation
+// of merkle roots inclusion in the longest chain.
+type merkleRootConfirmation struct {
+	Hash       string `json:"blockhash"`
+	MerkleRoot string `json:"merkleRoot"`
+	Confirmed  bool   `json:"confirmed"`
+}
+
+// merkleRootsConfirmationsResponse is an API reponse for confirming
+// merkle roots inclusion in the longest chain.
+type merkleRootsConfirmationsResponse struct {
+	AllIncluded   bool                     `json:"allIncluded"`
```

Comment:
### minor

maybe `AllConfirmed` to be consitant with `Confirmations` sturcts?

Reply:
Right, done